docs/README.md

```diff
@@ -26,7 +26,7 @@ React Resource Router was developed by Atlassian for [Jira](https://www.atlassia
 Resources describe and provide the data required for your route. This data is safely stored and accessed via the [`useResource`](api/hooks.md#use-resource) hook or [`ResourceSubscriber`](api/components.md#resourcesubscriber) component.
 
 ```js
-import { createResource } from 'react-resource-router';
```

Comment:
How does this get released? when I merge PR or when I release new npm version?

Reply:
I think when you merge to master

Reply:
@MonicaOlejniczak do you think we should update docs in another PR? to be sure it's not going to get public until the new version is released?

Reply:
Either is fine